frontend/src/v5/ui/v4Adapter/overrides/cards/compare.overrides.ts

```diff
@@ -35,4 +35,10 @@ export default css`
 	${FilterPanel} .MuiInputBase-root {
 		padding-left: 39px;
 	}
+
+	${Name} {
+		max-width: 317px;
+		overflow: hidden;
+		text-overflow: ellipsis;
```

Comment:
I think this needs a `white-space: nowrap;`
![image](https://user-images.githubusercontent.com/62481250/211303756-60108f17-0011-4d93-add6-397604f6d3f5.png)